front/src/routes/integration/all/mqtt/device-page/style.css

```
@@ -2,17 +2,458 @@
+  border-color: rgba(255, 255, 255, 0.12);
+}
+
+:global(.dark-mode) .backLink {
+  color: rgba(255, 255, 255, 0.65);
+}
+
+:global(.dark-mode) .backLink:hover {
+  color: #9ec5fe;
+}
+
+:global(.dark-mode) .featureCatalogCardDescription,
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)